src/config/ai-providers.ts

```
@@ -0,0 +1,223 @@
+export const getAIConfig = (): AIConfig | null => {
+  const provider = getCurrentProvider();
+
+  switch (provider) {
+    case 'azure':
+      const azureConfig = {
+        provider: 'azure' as const,
+        endpoint: import.meta.env.VITE_AZURE_OPENAI_ENDPOINT,
+        apiKey: import.meta.env.VITE_AZURE_OPENAI_API_KEY,
+        deploymentName: import.meta.env.VITE_AZURE_OPENAI_DEPLOYMENT_NAME,
+        apiVersion: import.meta.env.VITE_AZURE_OPENAI_API_VERSION || '2024-02-15-preview',
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)